### PR TITLE
fixed ellipsis character after menu items in UI 

### PR DESCRIPTION
**Pull request by RodyLipson**
## References
I made a few changes regarding #9874. Most of the menu items already had the correct ellipsis Unicode character. I fixed the ones that did not. 

## User-facing changes
Screen Shot 2021-03-24 at 11 13 44 AM
Screen Shot 2021-03-24 at 11 34 08 AM

### CODE REVIEW

File: packages/docmanager-extension/src/index.ts [hunk 1]
```diff
@@ -783,8 +783,7 @@ function addLabCommands(
   });
 
   commands.addCommand(CommandIDs.rename, {
-    label: () =>
-      trans.__('Rename %1…', fileType(contextMenuWidget(), docManager)),
+    label: () => trans.__('Rename…', fileType(contextMenuWidget(), docManager)),
```
Comment (jasongrout) [inline packages/docmanager-extension/src/index.ts:786]: It looks like this is missing a %1, which is important (that is the placeholder that is substituted by the file type). Can you put back in that `%1`?

Comment (jasongrout) [reply packages/docmanager-extension/src/index.ts:786]: Actually, it looks like this line didn't need to be touched?

Comment (RodyLipson) [reply packages/docmanager-extension/src/index.ts]: Hi I can put the %1 back in. There is a space after the rename menu item before the ellipsis. You can see the space in the UI. The reason I changed that line was to get rid of the space.

File: packages/docmanager-extension/src/index.ts [hunk 2]
```diff
@@ -784,7 +784,7 @@ function addLabCommands(
 
   commands.addCommand(CommandIDs.rename, {
     label: () =>
-      trans.__('Rename %1…', fileType(contextMenuWidget(), docManager)),
+      trans.__('Rename%1…', fileType(contextMenuWidget(), docManager)),
```
Comment (jasongrout) [inline packages/docmanager-extension/src/index.ts:787]: Sorry, we still need that space before the `%1` too :).

Comment (RodyLipson) [reply packages/docmanager-extension/src/index.ts:787]: No problem I'll put it back in now.

Comment (jasongrout) [reply packages/docmanager-extension/src/index.ts:787]: Ah, now I understand why you took it out. If the file type is empty, you end up with that extra space. How about leaving it as `Rename%1`, and put the space in the replacement text? Like this:
```suggestion
      trans.__('Rename%1…', ' ' + fileType(contextMenuWidget(), docManager)),
```

Comment (jasongrout) [reply packages/docmanager-extension/src/index.ts:787]: (You can see the effect by opening a notebook, say, and then looking at the menu item. Then it should say something like "Rename Notebook..."

Comment (RodyLipson) [reply packages/docmanager-extension/src/index.ts]: I see what you mean. I believe I updated this to match your suggested change :)

File: packages/docmanager-extension/src/index.ts [hunk 3]
```diff
@@ -784,7 +784,7 @@ function addLabCommands(
 
   commands.addCommand(CommandIDs.rename, {
     label: () =>
-      trans.__('Rename %1…', fileType(contextMenuWidget(), docManager)),
+      trans.__('Rename%1…', ' ' + fileType(contextMenuWidget(), docManager)),
```
Comment (jasongrout) [inline packages/docmanager-extension/src/index.ts:787]: Oh wait, I guess my suggested change always puts the space in there anyway. We really only want the space there if that fileType method returns a nonempty string. How about something like this, which only adds the space if needed?

label: () => {
  const t = fileType(contextMenuWidget(), docManager));
  if (t) {
    t = ' ' + t;
  }
  return trans.__('Rename%1…', t)
},

Comment (RodyLipson) [reply packages/docmanager-extension/src/index.ts]: Thanks I included this :) I just had to change const to let in order for it to compile properly

**Comment by jasongrout** [comment]
Thanks!

**Pull request merged**